src/common/utils.js

```diff
@@ -100,10 +103,41 @@ export function copyToClipboard(data) {
   textField.remove();
 };
 
+/* Returns date set to local timezone and in format of locale */
+export function getLocalTimeFormat(date, isUTC, timezone) {
+  var localDate = getLocalTime(date, isUTC, timezone);
+  return localDate.format("l") + " " + localDate.format("LTS");
+}
+
+/* Returns date set to local timezone and formatted in standard format of YYYY-MM-DDTHH:mm:ss */
+export function getStandardTimeFormat(date, isUTC, timezone) {
+  var localDate = getLocalTime(date, isUTC, timezone);
+  return localDate.format("YYYY-MM-DDTHH:mm:ss");
```

Comment:
Two small things 
- Use single quotes instead of double
- Move the actual string to the config file under common/ and import the config here